src/lightning/pytorch/core/module.py

```diff
@@ -451,7 +451,7 @@ def log(
                 " but it should not contain information about `dataloader_idx`"
             )
 
-        value = apply_to_collection(value, (Tensor, numbers.Number), self.__to_tensor, name)
```

Comment:
Mote: I don't think `__to_tensor` is used anywhere else now? Should I remove it?

Reply:
You could put your new implementation into `__to_tensor`, there's no need for a new method

Reply:
kk, was just worried it might surprise some folks